loopy/target/numba.py

```diff
@@ -103,6 +103,16 @@ def get_host_ast_builder(self):
     def get_device_ast_builder(self):
         return NumbaJITASTBuilder(self)
 
+    def get_kernel_executor_cache_key(self, *args, **kwargs):
+        return 'default'
+
+    def get_kernel_executor(self, knl, *args, **kwargs):
+        from loopy import generate_code
+        code, _ = generate_code(knl)
+        ns = {}
+        exec(code, ns)
+        return ns[knl.name]
```

Comment:
* This incurs a parse cost every time something is run. You might want to generate a function (rather than just a blob of code) and cache that function handle in your "executor".
* It ignores `*args` and `**kwargs`. Generating  a function would provide a clean place to pass those to.